ee/debug/shipper/shipper.go

```diff
@@ -268,6 +268,7 @@ func enrollSecret(k types.Knapsack) string {
 		return k.EnrollSecret()
 	}
 
+	// TODO this will need to respect the identifier when determining the secret file location for dual-launcher installations
 	b, err := os.ReadFile(launcher.DefaultPath(launcher.SecretFile))
```

Comment:
oooo.... This one we might need to fix. Can we shift it to using the secret in something configured instead of DefaultPath?

Reply:
I think we can make this fallback to reading from `knapsack.EnrollSecretPath()` first and that will fix this when running in situ at least. fixing for CLI triggered flares looks tougher